airflow/security/permissions.py

```diff
@@ -32,6 +32,7 @@
 RESOURCE_JOB = "Jobs"
 RESOURCE_POOL = "Pools"
 RESOURCE_PLUGIN = "Plugins"
+RESOURCE_TEMPLATE_FIELD = "TemplateFields"
```

Comment:
```suggestion
RESOURCE_TEMPLATE_FIELD = "Template Fields"
```
We've standardized around making the resource names human readable, since they are user facing.

Can you also place this alphabetically in the list of resources?

Reply:
Fixed!